recipes/glib/all/conandata.yml

```diff
@@ -14,3 +14,7 @@ sources:
   "2.66.0":
     url: "https://download.gnome.org/sources/glib/2.66/glib-2.66.0.tar.xz"
     sha256: "c5a66bf143065648c135da4c943d2ac23cce15690fc91c358013b2889111156c"
+patches:
+  "2.66.0":
```

Comment:
```suggestion
  "2.65.0": []
  "2.65.1": []
  "2.65.2": []
  "2.65.3": []
  "2.66.0":
```

Reply:
I made the patch available since 2.65.1 because it works from there. On 2.65.0 we keep the old behaviour.